plugin/storage/integration/integration.go

```diff
@@ -36,7 +36,7 @@ import (
 )
 
 const (
-	iterations = 30
+	iterations = 40
```

Comment:
go to 100

Reply:
Acked writes didn't seem to help unfortunately. I attempted to switch the acks to "local" as well as "all" to no avail. I'm guessing the ack never actually comes into effect because there is probably a timeout on the message transfer that is greater than the test time we have set. I ran kafkacat to see if the spans ever made it to the cluster and it seemed like the messages would always make it after some time.